src/xcode/ENA/ENA/Source/Client/__tests__/Mocks/ClientMock.swift

```diff
@@ -53,7 +53,7 @@ final class ClientMock {
 
 	var onGetTestResult: ((String, Bool, TestResultHandler) -> Void)?
 	var onSubmitCountries: ((_ payload: CountrySubmissionPayload, _ isFake: Bool, _ completion: @escaping KeySubmissionResponse) -> Void) = { $2(.success(())) }
-	var onGetRegistrationToken: ((String, String, Bool, @escaping RegistrationHandler) -> Void)?
+	var onGetRegistrationToken: ((String, String, String?, Bool, @escaping RegistrationHandler) -> Void)?
```

Comment:
Did I missed something in my PR?

Reply:
Not if you didn't need it yet, I just needed it for the Unit tests :)